log/models.go

```diff
@@ -0,0 +1,14 @@
+package log
+
+// StepStartedParams ...
+type StepStartedParams struct {
+	ExecutionId string `json:"uuid"`
+	Position    int    `json:"idx"`
+	IdVersion   string `json:"id_version"`
+	Id          string `json:"id"`
+	Version     string `json:"version"`
+	Title       string `json:"title"`
+	Collection  string `json:"collection"`
+	Toolkit     string `json:"toolkit"`
+	StartTime   string `json:"-"` // This value is only needed for the console logging.
```

Comment:
Won't we still need this for the log conversion (json->console)?

Reply:
No we do not because every json log will have a timestamp by default. We can use that time and print it as the step start time because the step execution starts after the step header is printed.